MSAL/src/instance/MSALAccountsProvider.m

```diff
@@ -363,17 +360,14 @@ - (void)allAccountsFromDevice:(MSALAccountEnumerationParameters *)parameters
         return;
     }
     
-    if (@available(macOS 10.15, *))
-    {
-        [self allAccountsFromSSOExtension:parameters
+    [self allAccountsFromSSOExtension:parameters
                         requestParameters:requestParameters
                           completionBlock:completionBlock];
-    }
 }
 
 - (void)allAccountsFromSSOExtension:(MSALAccountEnumerationParameters *)parameters
                   requestParameters:(MSIDRequestParameters *)requestParameters
-                    completionBlock:(MSALAccountsCompletionBlock)completionBlock API_AVAILABLE(ios(13.0), macos(10.15))
```

Comment:
Minimum version for iOS is now 14.0 (140000) and for macOS is 10.15 (101500) so all checks like this have been removed as they are now always true